sdk/log/logger.go

```diff
@@ -6,28 +6,66 @@ package log // import "go.opentelemetry.io/otel/sdk/log"
 import (
 	"context"
 
+	"go.opentelemetry.io/otel"
 	"go.opentelemetry.io/otel/log"
 	"go.opentelemetry.io/otel/log/embedded"
 	"go.opentelemetry.io/otel/sdk/instrumentation"
+	"go.opentelemetry.io/otel/trace"
 )
 
 // Compile-time check logger implements log.Logger.
 var _ log.Logger = (*logger)(nil)
 
 type logger struct {
 	embedded.Logger
+
+	provider             *LoggerProvider
+	instrumentationScope instrumentation.Scope
 }
 
 func newLogger(p *LoggerProvider, scope instrumentation.Scope) *logger {
-	// TODO (#5061): Implement.
-	return &logger{}
+	return &logger{
+		provider:             p,
+		instrumentationScope: scope,
+	}
 }
 
 func (l *logger) Emit(ctx context.Context, r log.Record) {
-	// TODO (#5061): Implement.
+	newRecord := l.newRecord(ctx, r)
+	for _, p := range l.provider.processors {
+		if err := p.OnEmit(ctx, newRecord); err != nil {
+			otel.Handle(err)
+		}
+	}
 }
 
-func (l *logger) Enabled(ctx context.Context, r log.Record) bool {
-	// TODO (#5061): Implement.
+func (l *logger) Enabled(context.Context, log.Record) bool {
 	return true
```

Comment:
This needs to check all of its processors and determine if any of them will actually be enabled prior to returning true.